tests/conftest.py

```diff
@@ -25,7 +25,7 @@
 from server.rating import RatingType
 from tests.utils import MockDatabase
 
-logging.getLogger().setLevel(logging.DEBUG)
+logging.getLogger().setLevel(TRACE)
```

Comment:
What that trace does? I didn't get it O_o

Reply:
It is used for logging the contents of every message that is sent/received by lobbyconnection.